integration_tests/CMakeLists.txt

```
@@ -703,6 +703,7 @@ RUN(NAME intrinsics_184 LABELS gfortran llvm) # selected_char_kind
 RUN(NAME intrinsics_185 LABELS gfortran llvm) # verify
 RUN(NAME intrinsics_186 LABELS gfortran llvm) # verify
 RUN(NAME intrinsics_187 LABELS gfortran llvm) # popcnt
+RUN(NAME intrinsics_parity LABELS gfortran llvm)
```

```suggestion
RUN(NAME intrinsics_188 LABELS gfortran llvm) #parity
```
Let us maintain the convention here